pkg/controller/backup_deletion_controller.go

```diff
@@ -442,6 +457,62 @@ func (c *backupDeletionController) deleteResticSnapshots(backup *v1.Backup) []er
 	return errs
 }
 
+func getCSIVolumeSnapshotsInBackup(backup *velerov1api.Backup, csiClient snapshotter.SnapshotV1beta1Interface, log *logrus.Entry) ([]snapshotv1beta1api.VolumeSnapshot, []error) {
+	csiVolSnaps := []snapshotv1beta1api.VolumeSnapshot{}
+	errs := []error{}
+
+	selector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			velerov1api.BackupNameLabel: label.GetValidName(backup.Name),
+		},
+	}
+	listOpts := metav1.ListOptions{
+		LabelSelector: metav1.FormatLabelSelector(&selector),
+	}
+
+	for _, ns := range pkgbackup.GetNamespacesInBackup(backup) {
```

Comment:
By using the informer/lister, I think this loop would be unnecessary.

See https://github.com/vmware-tanzu/velero/blob/master/pkg/cmd/server/server.go#L897-L899 and its use at 
https://github.com/vmware-tanzu/velero/blob/master/pkg/controller/backup_controller.go#L565

Reply:
Because the informer/lister wasn't made available in this controller chose to implement this loop.